pkg/store/bucket.go

```diff
@@ -775,10 +774,6 @@ func (s *BucketStore) Series(req *storepb.SeriesRequest, srv storepb.Store_Serie
 				mtx.Unlock()
 
 				return nil
-			}, func(err error) {
-				if err != nil {
```

Comment:
You are aware that this here made sure that `run.Group` was behaving like `errgroup`?